README.md

```diff
@@ -80,30 +83,6 @@ and then use them in your React components
 | Build package                         | `yarn build`            |
 | Run the component generator           | `yarn create-component` |
 
-### [Icons](./docs/ICONS.md)
-
-### [Publishing](./docs/PUBLISHING.md)
-
-## Working with the codebase
-
-Please refer to the following documentation to learn how to work with this codebase:
-
-- [Code guidelines](./docs/CODE_GUIDELINES.md)
-- [Tokens](./docs/TOKENS.md)
-- [Components](./docs/COMPONENTS.md)
-- [Typography](./docs/TYPOGRAPHY.md)
-- [Layout](./docs/LAYOUT.md)
-- [Icons](./docs/ICONS.md)
-
-## Workflow
-
-- [Contribution guidelines](./docs/CONTRIBUTING.md) - Please refer to these guidelines to learn how to contribute to the library.
-- [Release guidelines](./docs/PUBLISHING.md) - Please refer to these guidelines for instructions on how to publish a new version of the library.
-
-## Support
-
-For questions, feature requests, bugs, and more,feel free to reach out in [#help-edu-design-system](https://chanzuckerbergteam.slack.com/archives/CTFV79JH4) or [office hours](https://www.google.com/url?q=https://docs.google.com/spreadsheets/d/1zZguiMQHQLANjfUF-LjmPkbZ29I7ZXfl8TRDAhqDL0o/edit&sa=D&source=calendar&ust=1617083817378000&usg=AOvVaw2MJp29FMPv2AD1WJFX5Q2x) if you have additional questions (_note: these resources can only be accessed by CZI employees_).
-
```

Comment:
Guessing you're removing the Support section since it's not applicable to open source (which makes sense). Wondering if we should
- Keep a link to Publishing instructions
- Add a link to Zeroheight? Even though it's a CZI-specific resource, seems the cross-discoverability is useful to the team

Reply:
@anniehu4 for publishing, i added a link under "development" for that one. It's a little higher up now.

Good idea; adding a link to ZH here